test/lit/passes/outlining.wast

```diff
@@ -509,25 +692,7 @@
 ;; Test outlining works with call_indirect
 (module
   (table funcref)
-  ;; CHECK:      (type $0 (func))
-
-  ;; CHECK:      (type $1 (func (param i32 i32)))
-
-  ;; CHECK:      (table $0 0 funcref)
-
-  ;; CHECK:      (func $outline$ (type $0)
-  ;; CHECK-NEXT:  (call_indirect $0 (type $1)
-  ;; CHECK-NEXT:   (i32.const 0)
-  ;; CHECK-NEXT:   (i32.const 1)
-  ;; CHECK-NEXT:   (i32.const 2)
-  ;; CHECK-NEXT:  )
-  ;; CHECK-NEXT: )
-
-  ;; CHECK:      (func $a (type $0)
-  ;; CHECK-NEXT:  (call $outline$)
-  ;; CHECK-NEXT:  (call $outline$)
-  ;; CHECK-NEXT: )
-  (func $a
+  (func
```

Comment:
It looks like the check lines were removed because this function is no longer named. Was that intentional?

Reply:
The lines were reverted back to what's on main.